docs/fs.md

```diff
@@ -6,6 +6,12 @@ directory from the host with the guest.
 __virtio-fs__, also known as __vhost-user-fs__ is a virtual device defined by
 the VIRTIO specification which allows any VMM to perform filesystem sharing.
 
+## Limitations
+
+Using __virtio-fs__ devices in combination snapshot/restore is currently not supported.
+
+The reason is that virtio-fs doesn't support migration. And more specifically, the virtiofsd daemon described below doesn't implement the migration mechanisms that are exposed by vhost-user. Only DPDK and SPDK (vhost-user-net and vhost-user-block respectively) implement the migration mechanism.
+
```

Comment:
I think it is more logical to put this section at the end of the documentation. Please also wrap line at 80 characters.

Reply:
Let me add some more context and rationale for the proposed change.

You've made commits to this repo. You've already formed a mental model of how the concepts relate to each other.

I came to the docs as a newcomer, trying to build up a working mental model. That involves following the steps in the docs, and I'd venture that people do that from the top, following along with the steps one by one, trying to get things working. I ended up wasting time on virtio-fs because the limitations came after the steps to use it.

Put another way, I think it's very hard for project contributors to review the docs from a newcomer's perspective. It's hard to know what you know.